docs/includes/apiargs-MongoDBDatabase-method-createCollection-option.yaml

```diff
@@ -35,6 +35,20 @@ pre: |
   </reference/bson-type-comparison-order/#collation>` for the collection.
 ---
 arg_name: option
+name: expireAfterSeconds
+type: integer
+description: |
+  Available on 5.0+ servers only
+
+  Allows specifying a TTL after which documents will be removed from a
+  time-series collection.
+interface: phpmethod
+operation: ~
+optional: true
+post: |
+  .. versionadded:: 1.9
```

Comment:
`post:` is generally only helpful if we're inheriting common docs. In this case, it'd be more consistent with existing docs to just append this after two line breaks at the end of `description`.